network-store-server/src/main/resources/db/changelog/changesets/tapchanger_and_svc_regulation_mode_migration_20250704T121439Z.sql

```diff
@@ -0,0 +1,14 @@
+-- phase tap changer load tap changing capabilities
+UPDATE twowindingstransformer
+    SET phasetapchangerloadtapchangingcapabilities = true
+WHERE phasetapchangertapposition is not null or phasetapchangerlowtapposition is not null or phasetapchangertargetdeadband is not null or phasetapchangerregulationvalue is not null ;
+
+-- phase tap changer regulation
+UPDATE regulatingpoint
+set regulationmode = 'CURRENT_LIMITER', regulating = false
+where regulationmode = 'FIXED_TAP' and regulatingequipmenttype = 'TWO_WINDINGS_TRANSFORMER' and regulatingtapchangertype = 'PHASE_TAP_CHANGER';
```

Comment:
and regulatingequipmenttype = 'THREE_WINDINGS_TRANSFORMER'

Reply:
done